### PR TITLE
remove Known Issues Tags

### PR DESCRIPTION
**Pull request by asanzgom**

### CODE REVIEW

File: ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422__model_serving_llm_UI.robot [hunk 1]
```diff
@@ -267,7 +267,7 @@ Verify User With Edit Permission Can Deploy Query And Delete A LLM
     [Documentation]    This test case verifies that a user with Edit permission on a DS Project can still deploy, query
     ...    and delete a LLM served with caikit
     ...    ProductBug: https://issues.redhat.com/browse/RHOAIENG-548
```
Comment (FedeAlonso) [inline ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422__model_serving_llm_UI.robot:269]: @asanzgom could you please delete the Productbug links from the doc of the test case?

File: ods_ci/tests/Tests/500__jupyterhub/custom-image.robot [hunk 2]
```diff
@@ -97,7 +97,6 @@ Test Image From Local registry
     ...       ProductBug - https://github.com/opendatahub-io/odh-dashboard/issues/2185
```
Comment (FedeAlonso) [inline ods_ci/tests/Tests/500__jupyterhub/custom-image.robot:97]: Same

Comment (jstourac) [reply ods_ci/tests/Tests/500__jupyterhub/custom-image.robot:97]: Interesting on this one is that it hasn't been fixed yet the test started to work? :thinking:

File: ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422__model_serving_llm_UI.robot (hunk 1 above)
Comment (jstourac) [reply ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422__model_serving_llm_UI.robot:269]: this has been fixed in 2.6 already... wonder why it wasn't been removed from known issues that time...

Regarding the links to the reported issues - I slightly keen to keep them here for reference, just to remove the `ProductBug` at the beginning. Something like:

```suggestion
    ...    Issue reporated for this test in the past: https://issues.redhat.com/browse/RHOAIENG-548
```

but I can live with removal also

Comment (asanzgom) [reply ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422__model_serving_llm_UI.robot:269]: sure I will

**Review by jstourac — APPROVED** [review]

**Review by FedeAlonso — APPROVED** [review]

**Pull request merged**